Explorer/Assets/Scripts/ECS/StreamableLoading/Textures/GetTextureIntention.cs

```diff
@@ -12,19 +12,24 @@ public struct GetTextureIntention : ILoadingIntention, IEquatable<GetTextureInte
         public bool IsReadable;
         public TextureWrapMode WrapMode;
         public FilterMode FilterMode;
+        public bool IsVideoTexture;
```

Comment:
Adding these fields here smells a bit: they don't participate in the actual logic of texture loading -> they are only for stored data comparison.
Can you revisit this part? It feels like these fields should not be a part of this component

Reply:
good catch 💪 Indeed, it was redundant (probably some leftover from early solutions)